lang/main.json

```diff
@@ -309,8 +309,9 @@
         "tokenAuthFailed": "Sorry, you're not allowed to join this call.",
         "displayNameRequired": "Display name is required",
         "enterDisplayName": "Please enter your display name",
-        "extensionRequired": "Extension required:",
-        "firefoxExtensionPrompt": "You need to install a Firefox extension in order to use screen sharing. Please try again after you <a href='__url__'>get it from here</a>!",
+        "extensionRequired": "Extension required",
+        "firefoxScreenshareExtensionRequired": "You need to install a Firefox extension in order to use screen sharing. Please try again after installing the extension.",
```

Comment:
The problem when we change string keys is that all the translations will be lost. I see we're also changing the text here, so it's maybe worth it and it will look broken with the translated version anyway, but just mentioning if you feel like there's another way.

Reply:
There are multiple ways to approach this problem. The way I chose was to keep the body of notifications having only text. As such, leaving the string alone would cause the html to show up. If you really want it I can keep changing things so that the link can be in the body.